frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -27,8 +27,8 @@ export type InventoryLayerProps = {
   maxZoom?: number;
   /** Search filter to apply to properties. */
   filter?: IGeoSearchParams;
-  /** What to do when the point feature is clicked. */
-  onMarkerClick?: (point: PointFeature, position?: [number, number]) => void;
+  /** What to do when the marker is clicked. */
+  onMarkerClick: () => void;
```

Comment:
This wasn't being used at all (as far as I could tell) so I changed it

Reply:
Yes, I think it was used back when we had a completely different page when submitting properties, so this is fine.